Common/Util/WorkerThread.cs

```diff
@@ -68,7 +68,11 @@ public WorkerThread()
                     // pass, when the token gets cancelled
                 }
             })
-            { IsBackground = true, Name = "Isolator Thread" };
+            {
+                IsBackground = true,
+                Name = "Isolator Thread",
+                Priority = ThreadPriority.Highest
```

Comment:
Removing the priority doesn't explain much of the local gains. The idea of the change was to avoid the worker threads to take any CPU from the algorithm thread. Sadly cloud tests have a bigger noise, I mainly executed that test to take a look at the ram usage (runtime in seconds is present). 
**Test 7:**
```
public override void Initialize()
{
    SetStartDate(2017, 6, 1);
    SetEndDate(2018, 1, 1);
    foreach (var symbol in Symbols.Equity.All.Take(400))
    {
        AddData<TiingoNews>(AddSecurity(SecurityType.Equity, symbol, Resolution.Daily).Symbol);
    }
}
```
Without any priority
57.64 seconds at 333k data points per second. Processing total of 19,195,226 data points.
57.98 seconds at 331k data points per second. Processing total of 19,195,228 data points.
Workers low priority and algorithm thread high priority
57.14 seconds at 336k data points per second. Processing total of 19,195,223 data points.
57.57 seconds at 333k data points per second. Processing total of 19,195,228 data points.